test/conformance/testdata/ineligible_endpoints.yaml

```diff
@@ -109,6 +109,9 @@
 - endpoint: getInternalApiserverAPIGroup
   reason: Not eligible for conformance yet
   link: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190802-dynamic-coordinated-storage-version.md
+- endpoint: getResourceAPIGroup
```

Comment:
this makes sense until the APIs in this group go to GA

as an aside, there's other items in this list I'm surprised are exempted. a few examples:

* `*CoreV1Node` (these do *not* use the kubelet API as cited)
* `*StorageV1CSINode` / `*StorageV1CSIDriver` / `*StorageV1StorageClass` / `getStorageAPIGroup` (these allow setup of non-vendor-specific drivers... they should be able to be relied on across clusters)
* `createCoreV1NamespacedServiceAccountToken` (has not been optional since 1.21)
* `*CoreV1NamespacedPodEphemeralcontainers` (has been GA since 1.25)